lib/ansible/cli/__init__.py

```diff
@@ -8,6 +8,16 @@
 import locale
 import os
 import sys
+import argparse
+
+import typing
+if typing.TYPE_CHECKING:
+    from typing import Any, Dict, Optional, Union, Callable
```

Comment:
You can also replace `typing.Dict[...]` with `dict[...]`, `typing.Optional[<SOME TYPE>]` with `<SOME TYPE> | None`, `Union[<SOME TYPE>, <ANOTHER_TYPE>]` with `<SOME TYPE> | <ANOTHER TYPE>`, and `typing.Callable[...]` with `collections.abc.Callable[...]`.

```suggestion
    from collections.abc import Callable
    from typing import Any
```